packages/database/src/migrations/1739970132205-addAuditTrigger.ts

```diff
@@ -20,25 +21,25 @@ export async function up(query: QueryInterface): Promise<void> {
         record_id,
         record_update,
         record_data
-      ) VALUES (
-        TG_RELID,                 -- table_oid
-        TG_TABLE_SCHEMA,          -- table_schema
-        TG_TABLE_NAME,            -- table_name
-        CURRENT_TIMESTAMP,        -- logged_at
-        NEW.created_at,           -- created_at
-        NEW.updated_at,           -- updated_at
-        NEW.deleted_at,           -- deleted_at
-        NEW.updated_at_sync_tick, -- updated_at_sync_tick
-        NEW.updated_by_user_id,   -- updated_by_user_id
-        NEW.id,                   -- record_id
-        TG_OP = 'UPDATE',         -- record_update
-        to_jsonb(NEW.*)           -- record_data
-      );
-      RETURN NEW;
-    `,
+        ) VALUES (
+          TG_RELID,                 -- table_oid
+          TG_TABLE_SCHEMA,          -- table_schema
+          TG_TABLE_NAME,            -- table_name
+          CURRENT_TIMESTAMP,        -- logged_at
+          NEW.created_at,           -- created_at
+          NEW.updated_at,           -- updated_at
+          NEW.deleted_at,           -- deleted_at
+          NEW.updated_at_sync_tick, -- updated_at_sync_tick
+          coalesce(current_setting('${AUDIT_USERID_KEY}', true), uuid_nil()::text), -- updated_by_user_id
```

Comment:
```suggestion
          coalesce(nullif(current_setting('${AUDIT_USERID_KEY}', true), '') uuid_nil()::text), -- updated_by_user_id
```

This ensures that we default to uuid_nil even if the setting's value is not null, but is the empty string... as this is what happens when `set_config('foo', null)` is called.